MultimodalQnA/Dockerfile

```diff
@@ -13,7 +13,8 @@ RUN useradd -m -s /bin/bash user && \
     chown -R user /home/user/
 
 WORKDIR /home/user/
-RUN git clone https://github.com/opea-project/GenAIComps.git
+#RUN git clone https://github.com/opea-project/GenAIComps.git
+RUN git clone https://github.com/mhbuehler/GenAIComps.git --single-branch --branch dina/image_query
```

Comment:
This is a temporary change for testing. When my branch merges this can change to `mmqna-image-query` until the upstream PR is ready to merge. Although the gateway code moved to GenAIExamples, I still have a small change in the GenAIComps core that this would need [here](https://github.com/mhbuehler/GenAIComps/pull/12/files#diff-5b8b6e3ffcc8fdfd6bf6ed4fb7ecc47b03ff93a89850d627c25685267fc92d05R281) to allow multiple images to be sent to the LVM. The MMQnA megaservice and OPEA orchestrator code needs this because follow up queries are sent directly to the LVM and that could involve multiple images in the conversation.